Exercises/1-remove.js

```diff
@@ -1,7 +1,8 @@
 'use strict';
 
 const removeElement = (array, item) => {
-  // Remove item from array modifying original array
+  const index = array.indexOf(item);
+  index > -1 ? array.splice(index, 1) : array;
```

Comment:
Use if statement instead

Reply:
Thanks for the detailed response.
Done.